File: iOS/Demo/事件传递响应链/03-hitTest的原理/02-事件传递过程/ViewController.m
//
//  ViewController.m
//  02-事件传递过程
//
//  Created by czm on 2019/10/16.
//  Copyright © 2019 czm. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
}


- (void)viewDidAppear:(BOOL)animated {
    [super viewDidAppear:animated];
    
    for (UIView *v in self.view.subviews) {
        NSLog(@"======控制器View的顺序： %@", [v class]);
    }
}


@end

File: iOS/Demo/屏幕适配/layout/layout/ViewController.swift
//
//  ViewController.swift
//  layout
//
//  Created by czm on 2020/6/28.
//  Copyright © 2020 czm. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        navigationItem.title = "第一页"
        view.backgroundColor = .white

        
        
        print(UIScreen.main.bounds, UIScreen.main.scale, UIScreen.main.nativeScale)
        
    }
    
    
    
    @IBAction func clickAutoresizing(_ sender: UIButton) {
        navigationController?.pushViewController(AutoresizingViewController(), animated: true)
    }
    
    
    @IBAction func clickAutoLayout(_ sender: UIButton) {
        
        navigationController?.pushViewController(AutoLayoutViewController(), animated: true)
    }
    
    @IBAction func clickStackView(_ sender: UIButton) {
        navigationController?.pushViewController(StackViewController(), animated: true)
    }
    
    @IBAction func clickStackXib(_ sender: UIButton) {
        navigationController?.pushViewController(StackXibViewController(), animated: true)
    }
    
    
}

